ckanext/geodatagov/logic.py

```diff
@@ -470,7 +470,7 @@ def translate_spatial(old_spatial):
 
     # If we have 4 numbers separated by commas, transforme them as GeoJSON
     parts = old_spatial.strip().split(',')
-    if len(parts) == 4:
+    if len(parts) == 4 and all(x.replace('.', '', 1).isdigit() for x in parts):
```

Comment:
Why check `isdigit` instead of `float`?

Reply:
I see you pulled from https://stackoverflow.com/questions/354038/how-do-i-check-if-a-string-is-a-number-float. Personally I prefer the original post (see below), however I understand that the above is faster and contained in a single line. Maybe just put a comment noting why this is necessary, ie checking to make sure each part is a float (lat/long coordinate). That will help with the difficult reading of that line.
```
def is_number(s):
    try:
        float(s)
        return True
    except ValueError:
        return False
```